admin/requirements.txt

```diff
@@ -110,10 +101,10 @@ paramiko==2.4.2 \
     --hash=sha256:3c16b2bfb4c0d810b24c40155dbfd113c0521e7e6ee593d704e84b4c658a1f3b \
     --hash=sha256:a8975a7df3560c9f1e2b43dc54ebd40fd00a7017392ca5445ce7df409f900fcb \
     # via ansible
-prompt-toolkit==1.0.15 \
-    --hash=sha256:1df952620eccb399c53ebb359cc7d9a8d3a9538cb34c5a1344bdbeb29fbcc381 \
-    --hash=sha256:3f473ae040ddaa52b52f97f6b4a493cfa9f5920c255a12dc56a7d34397a398a4 \
-    --hash=sha256:858588f1983ca497f1cf4ffde01d978a3ea02b01c8a26a8bbc5cd2e66d816917
+prompt_toolkit==2.0.9 \
```

Comment:
was diff review performed here?

Reply:
Diff review has been performed (timeboxed to ~4 hr): https://github.com/freedomofpress/securedrop-debian-packaging/wiki/prompt_toolkit-1.0.15--2.0.9